.jshintrc

```diff
@@ -17,6 +17,7 @@
   "node": true,
   "expr": true,
   "unused": "vars",
+  "esversion": 6,
```

Comment:
PR already got LGT but I have realized about something...

What about test/.jshintrc file? Should it be modified the same way?

Reply:
Well, it is not really needed since ECMAScript 6 syntax is not used in the tests. Nevertheless, it does not hurt to add it. :)

Reply:
Fixed with 563d687b2b11004109478e9d3ea1a9fd084d7ded